### ISSUE TITLE
How to display the progress of the HTTP request in the lazy module

### ISSUE THREAD

**Issue by jiangyu19871119** [MurhafSousli/ngx-progressbar#65]
I want to use it for routing progress in the root module, and display the progress of the HTTP request in the lazy module.how can i do it?Thank you very much!

**Comment by jiangyu19871119** [comment]
AppComponent in the root module
```
  subscription: Subscription;

  constructor(private router: Router, private progressService: NgProgressService) {
  }

  ngOnInit(): void {
    this.subscription = this.router.events.subscribe(event => {
      if (event instanceof NavigationStart) {
        this.progressService.start();
      } else if (event instanceof NavigationEnd ||
        event instanceof NavigationCancel ||
        event instanceof NavigationError) {
        this.progressService.done();
      }
    }, (error: any) => {
      this.progressService.done();
    });
  }

  ngOnDestroy(): void {
    this.subscription.unsubscribe();
  }
```

HomeModule is the lazy module
```
import { HttpModule, BrowserXhr } from '@angular/http';
import { NgProgressModule, NgProgressBrowserXhr } from 'ngx-progressbar';
providers: [HomeService, { provide: BrowserXhr, useClass: NgProgressBrowserXhr }]
```

Works well in root module, but not in lazy module

**Comment by MurhafSousli** [comment]
Hi @jiangyu19871119  Try
```ts
export class LazyComponent {
    ngOnInit() {
        this.progressService.done();
    }
}
```

**Comment by jiangyu19871119** [comment]
Hi @MurhafSousli 

There are still some problems,
the progress bar does not go to the end and disappears.

**Comment by jiangyu19871119** [comment]
Can I use ```<ng-progress></ng-progress>``` both root module and inertia module at the same time?

**Comment by MurhafSousli** [comment]
@jiangyu19871119 Yes sure you can it both root module and lazy module at the same time, but you can only have one instace of `ng-progress`

 > There are still some problems,
the progress bar does not go to the end and disappears.

I can't figure out what you mean by 'there are still some problem', but if you can reproduce your problem in a plunker I can look into it

Here is a template for [auto-magic plnkr](https://plnkr.co/edit/LGTdFQCDBnT4wqlEAu5z?p=preview)
and another template for [start/stop plnkr](https://plnkr.co/edit/OEVjavH87Hk8GdAqdayK?p=preview)

**Comment by jiangyu19871119** [comment]
@MurhafSousli 
[the problem like this plnkr](https://plnkr.co/edit/giVaOM1x19RMKakneGB7?p=preview)
the progress bar does not go to the end and disappears.

**Comment by MurhafSousli** [comment]
@jiangyu19871119 turned out to be a dup of #66. it is fixed now in 2.0.7
